parasol/django.py

```diff
@@ -61,6 +61,11 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
                 raise ImproperlyConfigured('No URL in default SOLR_CONNECTIONS configuration')
 
             collection = default_solr.get('COLLECTION', '')
+
+            # if a CONFIGSET name is supplied, set it on SolrClient for use
+            # later, otherwise, default to 'basic_configs'
+            self.configSet = default_solr.get('CONFIGSET', 'basic_configs')
```

Comment:
Why store it on the solr client? Should only be used when auto-creating a core, right? Is that anywhere besides in the solr schema manage command?

Reply:
I thought about where to put it -- and it seemed sensible there for the index_command, and possibly something that might be useful in the future and helpful for the client to 'know'. I do use the functionality similarly in the unit testing.

If I were to pull it out, just have index pull the command from Django settings anyway and refactor tests similarly?